Formula/d/dagger.rb

```diff
@@ -29,6 +29,7 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/dagger/dagger/engine.Version=v#{version}
+      -X github.com/dagger/dagger/engine.Tag=v#{version}
```

Comment:
maybe also update test per this change?

Reply:
Ah, good point - currently there's no method that prints this out, but I can add that into dagger upstream.

However, if I add it in, it'll only work for releases past v0.12.4 - it won't work on v0.12.3. I'm not 100% sure of how homebrew handles this, is there a recommended practice?

Reply:
Let's just update it on the next version bump.